util/util.go

```diff
@@ -9,6 +9,10 @@ import (
 	"strings"
 )
 
+const (
+	OSRELEASE_PATH = "/etc/os-release"
```

Comment:
Ugh this is horrible, having this in `util` makes no sense. I think the code has got a little over-namespaced, provisioner is resin-specific for example so a 'resin' package makes no sense. And I don't think the helpers are so polluting the namespace as to need a separate package (which then faces issues with namespacing again.)

Also now we have a bunch of constants in one place and not the other.

I guess we can merge this for now and fix up these issues later @pcarranzav but we do have some technical debt here (also obv. we should be looking into non-hardcodey ways of specifying these things longer-term.)